lib/pymedphys/_mosaiq/connect.py

```diff
@@ -17,50 +17,28 @@
 """A toolbox for connecting to Mosaiq SQL.
 """
 
-from contextlib import contextmanager
-from getpass import getpass
+from pymedphys._imports import pymssql
 
-from pymedphys._imports import keyring, pymssql
-
-KEYRING_SCOPE = "PyMedPhys_SQLLogin_Mosaiq"
-
-
-def get_storage_name(hostname, port=1433, database="MOSAIQ"):
-    """returns the storage name for a given DB server + port + name
-
-    Parameters
-    ----------
-    hostname : str
-        db server name
-    port : int, optional
-        db server port, by default 1433
-    database : str, optional
-        db name, by default "MOSAIQ"
-
-    Returns
-    -------
-    str
-        the storage name to be used with keyring
-    """
-    return f"{KEYRING_SCOPE}_{hostname}:{port}/{database}"
-
-
-class WrongUsernameOrPassword(ValueError):
-    pass
+from . import credentials as _credentials
 
 
 def connect_with_credential(
-    sql_server_and_port, username, password, database="MOSAIQ"
+    username: str,
+    password: str,
+    hostname: str,
+    port: int = 1433,
+    database: str = "MOSAIQ",
```

Comment:
is the order of the new parameters ok?  it seems like it would be less confusing if the order of the new parameters were: hostname, port, username, password, database, because then they would be the same relative order as before.  or am I over-thinking it?

Reply:
> hostname, port, username, password, database, because then they would be the same relative order as before. or am I over-thinking it?

I agree that shooting for consistency is ideal. However, during PyMedPhys 0.*.* Beta stage if there are ways to unify the API across the library I have been prioritising that over invoking breaking changes.

I tried to build out a model of grouping "server parameters" which were consistent between many functions all at the end.